test/test_runner/test_runner.cpp

```diff
@@ -129,6 +150,19 @@ std::vector<std::string> TestRunner::convertResultToString(
     return actualOutput;
 }
 
+std::pair<std::string, uint64_t> TestRunner::convertResultToMD5Hash(QueryResult& queryResult) {
+    uint64_t numValuesHashed;
```

Comment:
Instead of comparing numValues, why not compare numTuples, since you are hashing on tuples? Comparing of num tuples looks simpler (for users) and consistent with what we are currently doing.

Reply:
Storing numTuples is probably the better option. It originally compared numValues because that's what SQLLogicTest does.

Reply:
I see. Do you think if we change it to comparing by numTuples, it will the transformation harder?

Reply:
It looks like in their tests, the number of values is equivalent to the number of tuples multiplied by the number of columns in the output, which shouldn't be hard to transform.